src/frequenz/dispatch/_dispatcher.py

```diff
@@ -221,26 +262,55 @@ def cancel(self, msg: str | None = None) -> None:
         for instance in self._actor_dispatchers.values():
             instance.cancel()
 
+    def is_dispatching(self, dispatch_type: str) -> bool:
+        """Check if the dispatcher is managing actors for a given dispatch type.
+
+        Args:
+            dispatch_type: The type of the dispatch to check.
+
+        Returns:
+            True if the dispatcher is managing actors for the given dispatch type.
+        """
+        return dispatch_type in self._actor_dispatchers
```

Comment:
I think failed dispatches should be considered as "dispatching" too, in general when querying this you want to know if `start_dispatching` was called on a type or not, at least according to the docs "if the dispatcher is managing actors for a given dispatch type". Maybe it can be renamed to `is_managed()` to make it less ambiguous?

If necessary, we can add a way to query if there are any actors successfully created to handle the type or not.

Reply:
TL;DR: All good, ignore this comment.

Oh, I mixed up the `actor_dispatcher` and the `dispatcher`. If the state of which actor instances are created successfully and which have failed, then I guess we can  leave this as it is, as the same `type` could have both (successfully running and failing), so there is no one simple answer.